lcclient/rpcclient.go

```diff
@@ -0,0 +1,33 @@
+package lcclient
+
+import (
+	"context"
+
+	"github.com/btcsuite/btcd/chaincfg/chainhash"
+	"github.com/btcsuite/btcd/wire"
+	"github.com/filecoin-project/go-jsonrpc"
+	"github.com/gonative-cc/relayer/bitcoinspv/types"
+)
+
+type Block struct {
+	Hash   *chainhash.Hash
+	Height int64
+}
+
+type Client struct {
```

Comment:
is this supposed to be the implementation of the interface? Its different from the interface

Reply:
To be changed in next PR, as per our discussion